src/components/Login/Login.js

```diff
@@ -185,6 +212,9 @@ const Login = ({ intl, location, dependencies: { dopplerLegacyClient, sessionMan
           </fieldset>
           <fieldset>
             <FormErrors />
```

Comment:
Right now, I am not sure how, but, I am pretty sure that you can _connect_ with the _form_ and take advantage of the current _FormErrors_ structure.

Something like what we are doing in _submit_: https://github.com/FromDoppler/doppler-webapp/blob/c2a39b262d18436fed9a77e245613d939087a680/src/components/Login/Login.js#L85-L87

Please, try to apply this approach or let me know, we should not have two different ways to do the same.

Reply:
fixed